src/pr-info.ts

```diff
@@ -283,23 +281,14 @@ function getReopenedDate(timelineItems: PR_repository_pullRequest_timelineItems)
     return lastItem?.createdAt && new Date(lastItem.createdAt);
 }
 
-type IssueComment = PR_repository_pullRequest_timelineItems_nodes_IssueComment;
-function getLastCommentishActivityDate(timelineItems: PR_repository_pullRequest_timelineItems, reviews: PR_repository_pullRequest_reviews | null) {
-
-    const lastIssueComment = findLast(timelineItems.nodes, (item): item is IssueComment => {
-        return item?.__typename === "IssueComment" && authorNotBot(item!);
-    });
-    const lastReviewComment = forEachReverse(reviews?.nodes, review => {
-        return findLast(review?.comments?.nodes, comment => !!comment?.author?.login)
-    });
-
-    if (lastIssueComment && lastReviewComment) {
-        return max([new Date(lastIssueComment.createdAt), new Date(lastReviewComment.createdAt)]);
-    }
-    if (lastIssueComment || lastReviewComment) {
-        return new Date((lastIssueComment || lastReviewComment)?.createdAt);
-    }
-    return undefined;
+function getLastCommentishActivityDate(prInfo: PR_repository_pullRequest) {
+    const latestIssueCommentDate = max(noNullish(prInfo.comments.nodes)
+        .filter(authorNotBot)
+        .map(comment => new Date(comment.createdAt)));
+    const latestReviewCommentDate = max(noNullish(prInfo.reviews?.nodes)
```

Comment:
Just a quick note: This should probably be updated in the future bot-review PR to ignore bot reviews too.